n3fit/src/n3fit/backends/keras_backend/base_layers.py

```diff
@@ -128,6 +128,7 @@ def apply_dense(xinput):
         {
             "input_shape": (1,),
             "replica_seeds": None,
+            "base_seed": 0,
```

Comment:
```suggestion
            "base_seed": 1,
```

maybe better start at one? Or would that be a problem somewhere down the line?